apps/golden-sample-app/src/app/services/journey-communication.service.ts

```diff
@@ -1,23 +1,19 @@
 import { Injectable } from '@angular/core';
 import { Router } from '@angular/router';
-import { BehaviorSubject } from 'rxjs';
-
-import {
```

Comment:
maybe you can check if we still need these interfaces. Otherwise you can remove them

Reply:
I did not understand what are the exact interfaces that you have mentioned here, can you point to lines or names, please.